zerver/integrations/nagios/service_notify.json

```diff
@@ -0,0 +1,3 @@
+{
+  "body": "**PROBLEM**: service is CRITICAL\n> CRITICAL - load average: 7.49, 8.20, 4.72"
+}
```

Comment:
We probably want `fixtures` in the name for this directory.  Maybe just `zerver/integration_fixtures/`.  Or `zerver/tests/fixtures/integrations`, which is a little off if we're not using these for tests, but might be close enough (and have the right properties, like being code we don't need to include in release tarballs).

Reply:
I think `zerver/integration_fixtures` might be better as these fixtures will be also used to generate screenshots.